modulles/news.modules.js

```diff
@@ -49,18 +57,35 @@ exports.selectCommentsByArticleId = (id) => {
   });
 };
 
-exports.getArticlesWithCommentCounts = () => {
-  return db.query(
-    `SELECT articles.article_id, articles.author , articles.title, articles.topic,
+exports.getArticlesWithCommentCounts = (queriesEntries) => {
+  let psqlStr = `SELECT articles.article_id, articles.author , articles.title, articles.topic,
+    articles.created_at, articles.votes, articles.article_img_url
+    , COUNT(comments.comment_id) as comment_count
+FROM articles
+FULL JOIN comments
```

Comment:
A LEFT join would work the best here - because the left table is articles, that would include articles both with and without comments.

The problem with a FULL JOIN, is that there's a risk of showing comments that no longer have an article. This could lead to rows with NULL values for article fields but some comment data, which is not ideal.

![image](https://github.com/MolchanovaA/news-api/assets/99129306/120920d5-6927-4820-99ac-f93a10ade8d7)

vs
![image](https://github.com/MolchanovaA/news-api/assets/99129306/dfa0d00e-0a9c-43ba-9dc7-69f751837101)